### DISCUSSION TITLE
Scatter Map visual cannot be exported to excel

### DISCUSSION THREAD

**Discussion by tinoptilogic** [RevealBi/Reveal.Sdk#142] [General]
When trying to export a scatter map to excel with the include visual selected, the visual doesn't get generated on the excel. Only the data that the scatter map is built on is written on excel.

**Comment by gkaveckis** [comment]
Hi

For the moment we are not supporting Scatter Map in excel exported files.
Please feel free to add it as a feature request here: https://github.com/RevealBi/Reveal.Sdk/issues

Regards.

**Comment by gkaveckis** [comment]
Closing as feature request was already submitted here: https://github.com/RevealBi/Reveal.Sdk/issues/144
